### PR TITLE
Fix msm tests

### PR DESCRIPTION
**Pull request by ulianaStashkiv**

### CODE REVIEW

File: applications/MachineStateManager/test/state_machine_test.cpp [hunk 1]
```diff
@@ -1,82 +1,41 @@
...
    std::unique_ptr<IStateFactoryMock> factoryMock{nullptr};
-   IMachineStateManagerMock* msmMock{nullptr};
+   std::unique_ptr<MachineStateClientMock> machineStateClientMock{nullptr};
    MsmStateFactory factory;
+   IMachineStateManagerMock* msmMock;
 };
 
 TEST_F(MsmStateMachineTest, ShouldInitCallEnter)
 {
+    EXPECT_CALL(*machineStateClientMock, Register(_,_)).WillOnce(Return(StateError::K_SUCCESS));
```
Comment (ihor96berizka) [inline applications/MachineStateManager/test/state_machine_test.cpp]: Add same test, but for unsuccessful registration(StateError:K_INVALID_STATE for example).

File: applications/MachineStateManager/src/machine_state_manager.cpp [hunk 2]
```diff
@@ -45,17 +45,17 @@ void MachineStateManager::transitToNextState(api::IAdaptiveApp::FactoryFunc next
...
 void MachineStateManager::reportApplicationState(ApplicationState state)
 {
-    m_appClient->ReportApplicationState(state);
+    m_appStateClient->ReportApplicationState(state);
 }
 
 void MachineStateManager::setMachineState(std::string state, int timeout)
 {
-  machineStateClient->SetMachineState(state, timeout);
+  m_machineStateClient->SetMachineState(state, timeout);
```
Comment (ihor96berizka) [inline applications/MachineStateManager/src/machine_state_manager.cpp]: SetMachineState expects uint32_t. Change timeout type: int -> std::uint32_t.

File: common/include/mocks.hpp [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+#include "gmock/gmock.h"
+
+using namespace MSM;
+
+using StateError = api::MachineStateClient::StateError;
+
+class IMachineStateManagerMock : public MachineStateManager
+{
+    public:
```
Comment (KateTheDev) [inline common/include/mocks.hpp]: Excess space before access specifier

File: common/src/i_machine_state_client_wrapper.cpp [hunk 4]
```diff
@@ -5,7 +5,7 @@ namespace api {
 
 using StateError = MachineStateClient::StateError;
 
-MachineStateClientWrapper::MachineStateClientWrapper() : m_machineClient{SOCKET_NAME}
+MachineStateClientWrapper::MachineStateClientWrapper() : m_machineClient{EM_SOCKET_NAME}
```
Comment (rBekesh) [inline common/src/i_machine_state_client_wrapper.cpp]: Should be IPC_PROTOCOL + EM_SOCKET_NAME

**Pull request merged**